### PR TITLE
Bug fix: added missing parameters to the functions

### PR DESCRIPTION
**Pull request by mrgunior**
Fixed a bug with the update instance test case. It was missing the service_id and plan_id, which for updating are required parameters. 

Operation was missing so I also added that:
When a value for operation is included with asynchronous responses for Provision, Update, and Deprovision requests, the Platform MUST provide the same value using this query parameter as a percent-encoded string. If present, MUST be a non-empty string

### CODE REVIEW

File: test/common/common.go [hunk 1]
```diff
@@ -8,6 +8,8 @@ import (
 	"reflect"
 	"time"
 
+	"github.com/antihax/optional"
+
```
Comment (BenElgar) [inline test/common/common.go]: Unnecessary new line

File: test/common/common.go [hunk 2]
```diff
@@ -8,6 +8,8 @@ import (
 	"reflect"
 	"time"
 
+	"github.com/antihax/optional"
```
Comment (BenElgar) [inline test/common/common.go:11]: Is this a library they already use?

File: test/common/update_instance.go [hunk 3]
```diff
@@ -40,8 +40,10 @@ func TestUpdateInstance(
 
 		if async {
 			Convey("should return 422 UnprocessableEntity if missing accepts_incomplete", func() {
+				tempBody := openapi.ServiceInstanceUpdateRequest{}
+				deepCopy(req, &tempBody)
```
Comment (BenElgar) [inline test/common/update_instance.go:44]: What's the point of this change?

Comment (mrgunior) [reply test/common/update_instance.go:44]: When provisioning an instance it is required to have service_id and plan_id to be non-empty values. This test case was missing this, which caused it to fail no matter what. The other functions do contain tempBody which in turn contains service_id and plan_id.

File: test/common/common.go (hunk 2 above)
Comment (mrgunior) [reply test/common/common.go:11]: ohh wait good question... I will double check this.

File: test/common/common.go (hunk 1 above)
Comment (mrgunior) [reply test/common/common.go]: Probably VC. Will push the changes

File: test/common/common.go (hunk 2 above)
Comment (mrgunior) [reply test/common/common.go:11]: Yes have a look at this [link](https://github.com/mrgunior/osb-checker/blob/dev/autogenerated/go-client/api_service_instances.go).

**Review by BenElgar — APPROVED** [review]

**Pull request merged**